cnd/src/main.rs

```diff
@@ -138,31 +143,35 @@ fn main() -> anyhow::Result<()> {
         Arc::clone(&ethereum_connector),
         lnd_connector_params,
         Arc::clone(&swap_communication_states),
+        Arc::clone(&rfc003_alpha_ledger_states),
+        Arc::clone(&rfc003_beta_ledger_states),
         Arc::clone(&alpha_ledger_states),
         Arc::clone(&beta_ledger_states),
         Arc::clone(&halight_states),
         &database,
         runtime.handle().clone(),
     )?;
 
-    let facade2 = Facade2 {
-        swarm: swarm.clone(),
-        alpha_ledger_states: Arc::clone(&alpha_ledger_states),
-        beta_ledger_states: Arc::clone(&halight_states),
-    };
-
+    // RCF003 protocol
     let deps = Facade {
         bitcoin_connector,
         ethereum_connector,
-        alpha_ledger_states,
-        beta_ledger_states,
+        alpha_ledger_states: Arc::clone(&rfc003_alpha_ledger_states),
+        beta_ledger_states: Arc::clone(&&rfc003_beta_ledger_states),
         swap_communication_states,
         swap_error_states,
         seed,
         db: database,
         swarm: swarm.clone(),
     };
 
+    // split protocols
+    let facade2 = Facade2 {
```

Comment:
```suggestion
    let split_protocols_facade = SplitProtocolsFacade {
```

Reply:
This is actually a good name, Or even better: rename `Facade` to `Rfc003Facade` if you feel keen.

Reply:
Cool, I like it.  Will use `Rfc003Facade` in line with the other renames.  thanks